widgets/GCSArgumentSetter-action.json

```diff
@@ -33,35 +33,35 @@
         },
         {
           "name": "serviceAccountType",
-          "label": "Provide Service Account",
+          "label": "Service Account Type",
           "widget-type": "radio-group",
           "widget-attributes": {
             "layout": "inline",
             "default": "filePath",
             "options": [
-              {
-                "id": "JSON",
-                "label": "JSON"
-              },
               {
                 "id": "filePath",
                 "label": "File Path"
+              },
+              {
+                "id": "json",
+                "label": "JSON"
               }
             ]
           }
         },
         {
           "widget-type": "textbox",
-          "label": "Service Account File Path",
+          "label": "File Path",
           "name": "serviceFilePath",
           "widget-attributes": {
             "default": "auto-detect"
           }
         },
         {
           "widget-type": "textbox",
-          "label": "Service Account JSON",
-          "name": "serviceAccountJSON"
+          "label": "JSON",
+          "name": "serviceJson"
```

Comment:
GCSArgumentSetter already had this feature and was merged to develop, since those code changes will be used by all other GC plugins we had to unify GCSArgumentSetter.

Reply:
Why are we changing name? I think it should be `serviceAccountJSON`